--- /share/applications/solaar.desktop
+++ /share/applications/solaar.desktop
@@ -5,7 +5,7 @@
 Comment[hr]=Upravitelj Logitechovih uređaja povezanih putem Unifying i Nano prijemnika
 Comment[ru]=Управление приёмником Logitech Unifying Receiver
 Comment[es]=Administrador de periféricos de Logitech Receptor Unifying
-Exec=solaar
+Exec=solaar --window=hide --battery-icons=symbolic
 Icon=solaar
 StartupNotify=true
 Terminal=false
